File: app/src/main/java/vn/edu/usth/minigh/fragments/IssuesListFragment.java
package vn.edu.usth.minigh.fragments;

import android.content.Intent;
import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.RadioGroup;
import android.widget.TextView;

import vn.edu.usth.minigh.DiscussionActivity;
import vn.edu.usth.minigh.IssueFragment;
import vn.edu.usth.minigh.R;

public class IssuesListFragment extends Fragment {
    Fragment frag;

    public IssuesListFragment() {
        super(R.layout.fragment_issues_list);
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_issues_list, container, false);
        RadioGroup sg = (RadioGroup) view.findViewById(R.id.segmented2);
        sg.check(R.id.button21);
        addFrag("Open", 3);
        sg.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
            @Override
            public void onCheckedChanged(RadioGroup group, int checkedId) {
                switch(checkedId){
                    default:
                        addFrag("Open", 3);
                        break;
                    case R.id.button22:
                        addFrag("Closed", 5);
                        break;
                }
            }
        });
        return view;
    }

    public void addFrag(String txt, int number){
        FragmentManager fm = getChildFragmentManager();
        frag = fm.findFragmentById(R.id.issuesFragment);
        FragmentTransaction  ft = fm.beginTransaction();
        frag = new IssueFragment(txt, number);
        ft.replace(R.id.issuesFragment, frag);
        ft.commit();
    }

    public void goToIssue(View view) {
        Intent intent = new Intent(getActivity(), DiscussionActivity.class);
        TextView title = (TextView) view.findViewById(R.id.issuePrContent);
        intent.putExtra("title", (String) title.getText());

        TextView description = (TextView) view.findViewById(R.id.issuePrContent);
        intent.putExtra("description",(String) description.getText());
        startActivity(intent);
    }
}

File: app/src/main/java/vn/edu/usth/minigh/fragments/IssuesListFragment.kt
package vn.edu.usth.minigh.fragments

import android.os.Bundle
import android.util.Log
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.RadioGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.lifecycleScope
import kotlinx.coroutines.launch
import vn.edu.usth.minigh.IssueFragment
import vn.edu.usth.minigh.R
import vn.edu.usth.minigh.api.Issue
import vn.edu.usth.minigh.api.github
import java.util.ArrayList

class IssuesListFragment : Fragment(R.layout.fragment_issues_list) {
    var frag: Fragment? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        val view = inflater.inflate(R.layout.fragment_issues_list, container, false)
        val sg = view.findViewById<View>(R.id.segmented2) as RadioGroup
        sg.check(R.id.button21)

        val repo_name = requireArguments().getString("repo name")!!

        lifecycleScope.launch {
            val issue = github.issueRepo(repo_name, "open")
            val igOpen:ArrayList<Issue> = issue
            val issueClose = github.issueRepo(repo_name,"closed")
            val igClose:ArrayList<Issue> = issueClose
            addFrag("Open", igOpen.count(), igOpen!!)
            sg.setOnCheckedChangeListener { group, checkedId ->
                when (checkedId) {
                    R.id.button22 -> addFrag("Closed", igClose.count(), igClose!!)
                    else -> addFrag("Open", igOpen.count(), igOpen!!)
                }
            }
        }
        return view
    }

    fun addFrag(txt: String?, number: Int, ig:ArrayList<Issue>) {
        val fm = childFragmentManager
        frag = fm.findFragmentById(R.id.issuesFragment)
        val ft = fm.beginTransaction()
        frag = IssueFragment.newInstance(txt, number, ig)
        ft.replace(R.id.issuesFragment, frag as IssueFragment)
        ft.commit()
    }
}